[KubeRay] Fix RAY_LOG_TO_STDERR YAML (#41164)
ray-project/ray: doc/source/cluster/kubernetes/user-guides/logging.md

(redirect-to-stderr)=
## Redirecting Ray logs to stderr

By default, Ray writes logs to files in the `/tmp/ray/session_*/logs` directory.
If your log processing tool is capable of capturing log records written to stderr, you can redirect Ray logs to the stderr stream of Ray containers by setting the environment variable `RAY_LOG_TO_STDERR=1` on all Ray nodes.

```{admonition} Alert: this practice isn't recommended.
:class: caution

> If `RAY_LOG_TO_STDERR=1` is set, Ray will not write logs to files.

Reviewer: ```suggestion
If `RAY_LOG_TO_STDERR=1` is set, Ray doesn't write logs to files.
```

Author: I just found out that Vale does not work for the admonition block.

> Consequently, this can cause some Ray features that rely on log files to malfunction.

Reviewer: ```suggestion
Consequently, this behavior can cause some Ray features that rely on log files to malfunction.
```

> For instance, {ref}`worker log redirection to driver <log-redirection-to-driver>` will not work.

Reviewer: ```suggestion
For instance, {ref}`worker log redirection to driver <log-redirection-to-driver>` doesn't work if you redirect Ray logs to stderr.
```

> If these features are required, consider using the {ref}`Fluent Bit solution <kuberay-fluentbit>` mentioned above.

Reviewer: ```suggestion
If you need these features, consider using the {ref}`Fluent Bit solution <kuberay-fluentbit>` mentioned above.
```

> For clusters on VMs, do not redirect logs to stderr. Instead, follow {ref}`this guide <vm-logging>` to persist logs.

Reviewer: ```suggestion
For clusters on VMs, don't redirect logs to stderr. Instead, follow {ref}`this guide <vm-logging>` to persist logs.
```

  :::

  :::{tab-item} KubeRay

>   Set environment variable `RAY_LOG_TO_STDERR` to `1` in the Ray container of each Ray Pod.

Reviewer: ```suggestion
  Set the `RAY_LOG_TO_STDERR` environment variable to `1` in the Ray container of each Ray Pod.
```

>   See this [example YAML file](https://gist.github.com/kevin85421/3d676abae29ebd5677428ddbbd4c8d74) as a reference.

Reviewer: ```suggestion
  Use this [example YAML file](https://gist.github.com/kevin85421/3d676abae29ebd5677428ddbbd4c8d74) as a reference.
```
